mussel

```diff
@@ -743,23 +765,26 @@ printf "PATH: \"$PATH\"\nMAKE: \"$MAKE\"\n\n" >>"$MLOG"
 printf "Host Kernel: \"$(uname -a)\"\nHost Info:\n$(cat /etc/*release)\n" >>"$MLOG"
 printf "\nStart Time: $(date)\n\n" >>"$MLOG"
 
+# if defined, c++ lang support is built in gcc
+[ $CXX_SUPPORT = "yes" ] && ENABLE_CXX_YES="abcd"
+
 # if defined, fortran lang support is built in gcc
 [ $FORTRAN_SUPPORT = "yes" ] && ENABLE_FORTRAN_YES="abcd"
 
 # ----- Prepare Packages ----- #
 printf "-----\nprepare\n-----\n\n" >>"$MLOG"
-mpackage binutils "$binutils_url" $binutils_sum $binutils_ver
-mpackage gcc "$gcc_url" $gcc_sum $gcc_ver
-mpackage gmp "$gmp_url" $gmp_sum $gmp_ver
-[ $ISL_SUPPORT = "yes" ] && mpackage isl "$isl_url" $isl_sum $isl_ver
+mpackage binutils "$binutils_url" "$binutils_sum" $binutils_ver
```

Comment:
I don't want to be _that_ type of shell guy but I must say that there's no need to quote checksums since we do not expect it to have spaces or anything --- I don't think it would be even a micro-optimzation, just a pure formality. I know you've unset the ``$linux_sum`` variable and made it empty, but it also shouldn't be this way.